src/components/User/User.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { userType } from '../../types';
+
+export const User = ({ name, email, address }) => (
+  <>
```

Comment:
don't overuse fragment, prefer to use semantic html elements

Reply:
Okay, done